src/components/layout/Footer.tsx

```diff
@@ -0,0 +1,18 @@
+const Footer = () => {
+    return (
+      <footer
+      role="contentinfo"
+      className="bg-background-footer text-text-primary py-8 w-full"
+    >
+      <div className="text-center">
+        <p>&copy; 2024 Font Hill Records. All rights reserved.</p>  
+        </div>
```

Comment:
```suggestion
        <p className="text-center">&copy; 2024 Font Hill Records. All rights reserved.</p>  
```
I think we can avoid using another div here with this solution - or else this styling can be applied to the footer generally. Until we get more content in the footer when a div might be useful lets keep it simple?

Reply:
Hey Max, thanks for the suggestion! I’ve removed the unnecessary <div>